server/repositories/cmsApi.js

```diff
@@ -0,0 +1,37 @@
+const { getEstablishmentUiId } = require('../utils');
+
+class CmsApi {
+  constructor(jsonApiClient) {
+    this.jsonApiClient = jsonApiClient;
+  }
+
+  async primaryMenu(prisonId = 0) {
+    const response = await this.jsonApiClient.get(
+      `/jsonapi/node/landing_page?fields[node--landing_page]=title,field_moj_description,drupal_internal__nid,field_moj_prisons`,
+    );
+    return this.transform(response?.data || {}, prisonId);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  transform(data, prisonId) {
+    const items = Object.values(data)
+      .filter(({ relationships }) => {
```

Comment:
this filter can go once  we move to the json api endpoints that allow you to specify a prison as this bit is all handled in drupal

Reply:
when this goes we can get rid of quite a bit of code